src/ui/settings/ViewAppPermissions.qml

```diff
@@ -121,8 +121,8 @@ Item {
                 //% "VPN protects all apps by default. Turn off to choose which apps Mozilla VPN should not protect."
                 sublabelText: qsTrId("vpn.settings.protectAllApps.description")
 
-                //% "VPN protects all apps by default. Turn off to choose which apps Mozilla VPN should not protect."
-                toolTipTitleText: qsTrId("vpn.settings.protectAllApps.description")
+                //% "Protect all apps with VPN"
```

Comment:
If this is a new string, please use `translations/strings.yaml`

Reply:
@bakulf We are reusing a string - not a new string. Is that okay?

Reply:
If you are reusing a string, do not add the comment at line 124. That comment, starting with `//%` is used by lupdate to get the initial string content.